controllers/api/campaign.go

```diff
@@ -127,7 +127,8 @@ func (as *Server) CampaignComplete(w http.ResponseWriter, r *http.Request) {
 	id, _ := strconv.ParseInt(vars["id"], 0, 64)
 	switch {
 	case r.Method == "GET":
-		err := models.CompleteCampaign(id, ctx.Get(r, "user_id").(int64))
+		userId := ctx.Get(r, "user_id").(int64)
```

Comment:
These changes aren't needed for this functionality - can we please revert them to keep this PR as focused as possible?